packages/std/src/encoding.rs

```diff
@@ -48,6 +48,37 @@ impl From<&[u8]> for Binary {
     }
 }
 
+// Macro needed until https://rust-lang.github.io/rfcs/2000-const-generics.html is stable.
+// See https://users.rust-lang.org/t/how-to-implement-trait-for-fixed-size-array-of-any-size/31494
+macro_rules! implement_from_for_fixed_length_arrays {
+    ($($N:literal)+) => {
+        $(
+            // Reference
+            impl From<&[u8; $N]> for Binary {
+                fn from(source: &[u8; $N]) -> Self {
+                    Self(source.to_vec())
+                }
+            }
+
+            // Owned
+            impl From<[u8; $N]> for Binary {
+                fn from(source: [u8; $N]) -> Self {
+                    // Implementation available for $N <= 32.
+                    // Eequires https://caniuse.rs/features/vec_from_array, avaiable since Rust 1.44.0.
```

Comment:
<nit> s/Eequires/Requires/

Reply:
Thank you. This was fixed in the final commit I manually merged at the end of the day (which Github never showed here because Github is broken today).